scripts/api-compare/api-compare.mjs

```diff
@@ -0,0 +1,350 @@
+import { Command } from 'commander'
+import { diffLinesUnified, diff } from 'jest-diff'
+import stripAnsi from 'strip-ansi'
+
+import fs from 'fs'
+import path from 'path'
+import { fileURLToPath } from 'url'
+
+const __filename = fileURLToPath(import.meta.url) // get the resolved path to the file
+const __dirname = path.dirname(__filename) // get the name of the directory
+
+const versionMetadataPath = path.resolve(
+	__dirname,
+	'../../app/api/versionMetadata.json'
+)
+const versionMetadata = JSON.parse(
+	fs.readFileSync(versionMetadataPath, 'utf-8')
+)
+
+const program = new Command()
+
+program
+	.option('-n, --new-api-url <url>', 'New API URL')
+	.option('-o, --old-api-url <url>', 'Old API URL')
+	.option('-v, --version <version>', 'Version')
+	.option('-p, --product <product>', 'Product')
+	.option(
+		'-a, --api <api>',
+		'API type: "content", "nav-data", "version-metadata", or "content-versions"',
+		'content'
+	)
+	.option(
+		'-d, --diff <type>',
+		'Diff type for API type "content": "metadata", "markdown", or "everything"',
+		'markdown'
+	)
+	.option('-r, --drop-keys <keys>', 'Result keys to drop', (value) =>
+		value.split(',').map((key) => key.trim())
+	)
+	.option('-t, --num-of-tests <number>', 'Number of tests', parseInt, 10)
```

Comment:
let's remove this and just test every route in a version and product that they give us.